puppet/modules/redmine/templates/postgresql_backup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Backs up the OpenShift PostgreSQL database for this application
+ 
+FILENAME="/usr/share/redmine_data/redmine.backup.sql.gz"
+
+touch $FILENAME
+chown postgres $FILENAME
```

Comment:
This shouldn't be needed since the pipe runs as root.

Reply:
Root has no ident access to the DB, so you need another user. We could use the Redmine user, put `postgres` is guaranteed to exist anyway...

Reply:
You can still sudo for pg_dump, just this chown line isn't needed I think.

Reply:
Oh right, I read the wrong thing. Done.